062-EntraIDB2B/Student/Challenge-05.md

```diff
@@ -0,0 +1,41 @@
+# Challenge 05 - Integrate your application with Entra ID
+
+[< Previous Challenge](./Challenge-04.md) - **[Home](../README.md)** - [Next Challenge >](./Challenge-06.md)
+
+
+## Introduction
+
+The code samples are built and maintained by Microsoft to demonstrate usage of our authentication libraries with the Microsoft identity platform. Common authentication and authorization scenarios are implemented in several application types, development languages, and frameworks.
+
+Each tutorial / walkthrough describes how to build the project (if applicable) and run the sample application. Comments in the code help you understand how these libraries are used in the application to perform authentication and authorization by using the identity platform.
```

Comment:
The walkthrough feel more prescriptive than what a WTH should be.  Is there a way to challenge the students more?